packages/driver/src/cy/commands/sessions/index.ts

```diff
@@ -515,4 +515,8 @@ export default function (Commands, Cypress, cy) {
   })
 
   Cypress.session = sessions
+  // Cypress.session is currently unsupported within the cy.origin() callback.
+  // This also stops us from using it internally, because we replace it with
+  // a proxy that throws an error. We can use this reference internally instead.
+  Cypress._session = sessions
```

Comment:
This change shouldn't attach more to the Cypress global. Seems like a good time to figure out the best structure for the get/set/clear cookies  & storage methods that live in this file. They don't need to be exposed globally on the sessions API anymore with the new commands.

Reply:
I moved the storage-related methods out, so there's no longer a need for `Cypress._session`. I'd rather not blow out the scope of this PR by touching any of the other methods. I'll extract the cookie-related ones when I work on the getAll/clearAll cookie commands.